### PR TITLE
Update Chinese localization

### PR DESCRIPTION
**Pull request by auroursa**
This PR includes the latest translations for zh-CN and zh-TW, and cleans up outdated strings.
cc @KuwaLee  @cirx1e

### CODE REVIEW

File: src/locale/locales/zh-TW/messages.po [hunk 1]
```diff
@@ -262,16 +262,20 @@ msgstr "30 天"
 msgid "7 days"
 msgstr "7 天"
 
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:266
+msgid "A virtual certificate with text \"Celebrating 10M users on Bluesky, #{0}, {displayName} {handle}, joined on {joinedDate}\""
+msgstr "虛擬證書，其中包含文字「慶祝 Bluesky 上的 1000 萬用戶，#{0}，{displayName} {handle}，於 {joinedDate} 加入」"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:267]:
```suggestion
msgstr "一張虛擬證書，上面寫著「慶祝 Bluesky 達到 1000 萬用戶，#{0}，{displayName} {handle}，於 {joinedDate} 加入」"
```

File: src/locale/locales/zh-TW/messages.po [hunk 2]
```diff
@@ -848,15 +852,20 @@ msgstr "Bluesky 是一個開放的網路，您可以自行挑選託管服務供
 msgid "Bluesky is better with friends!"
 msgstr "Bluesky 因朋友而更好！"
 
-#: src/components/dialogs/nuxs/TenMillion/index.tsx:206
+#: src/components/dialogs/nuxs/TenMillion/Trigger.tsx:43
+#: src/components/dialogs/nuxs/TenMillion/Trigger.tsx:59
+msgid "Bluesky is celebrating 10 million users!"
+msgstr "Bluesky 正在慶祝用戶突破 1000 萬！"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:858]:
```suggestion
msgstr "Bluesky 正在慶祝達到 1,000 萬用戶的里程碑！"
```

File: src/locale/locales/zh-TW/messages.po [hunk 3]
```diff
@@ -848,15 +852,20 @@ msgstr "Bluesky 是一個開放的網路，您可以自行挑選託管服務供
...
+
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:256
 msgid "Bluesky now has over 10 million users, and I was #{0}!"
-msgstr ""
+msgstr "Bluesky 現在擁有超過 1000 萬用戶，而我是第#{0}個！"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:862]:
```suggestion
msgstr "Bluesky 現在已經有超過 1,000 萬的用戶，而我是第 #{0} 位！"
```

File: src/locale/locales/zh-TW/messages.po [hunk 4]
```diff
@@ -1016,9 +1025,9 @@ msgstr "字幕（.vtt）"
 msgid "Captions & alt text"
 msgstr "字幕和替代文字"
 
-#: src/components/dialogs/nuxs/TenMillion/index.tsx:368
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:434
 msgid "Celebrating {0} users"
-msgstr ""
+msgstr "慶祝{0}用戶"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:1030]:
```suggestion
msgstr "慶祝達到 {0} 位用戶"
```

File: src/locale/locales/zh-CN/messages.po [hunk 5]
```diff
@@ -1016,9 +1025,9 @@ msgstr "字幕（.vtt）"
 msgid "Captions & alt text"
 msgstr "字幕及替代文本"
 
-#: src/components/dialogs/nuxs/TenMillion/index.tsx:368
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:434
 msgid "Celebrating {0} users"
-msgstr "庆祝第 {0} 位用户"
+msgstr "庆祝第{0}名用户"
```
Comment (cirx1e) [inline src/locale/locales/zh-CN/messages.po:1030]:
```suggestion
msgstr "庆祝拥有{0}名用户"
```

https://github.com/bluesky-social/social-app/blob/7e2456b906563464c8e43867e62f07df9109bc2b/src/components/dialogs/nuxs/TenMillion/index.tsx#L435
这部分在源码里是写死的，一千万用户

File: src/locale/locales/zh-TW/messages.po [hunk 6]
```diff
@@ -3114,6 +3141,14 @@ msgstr "邀請您的朋友跟隨您喜歡的動態源和人物"
 msgid "Invites, but personal"
 msgstr "邀請，但僅限個人"
 
+#: src/screens/Signup/StepInfo/index.tsx:77
+msgid "It looks like you may have entered your email address incorrectly. Are you sure it's right?"
+msgstr ""
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:3146]:
```suggestion
msgstr "您似乎輸入了錯誤的電子郵件地址，您確定這是正確的嗎？"
```

Update Translates

File: src/locale/locales/zh-TW/messages.po [hunk 7]
```diff
@@ -3114,6 +3141,14 @@ msgstr "邀請您的朋友跟隨您喜歡的動態源和人物"
...
+
+#: src/screens/Signup/StepInfo/index.tsx:238
+msgid "It's correct"
+msgstr ""
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:3150]:
```suggestion
msgstr "這是正確的地址。"
```

Update Translates

File: src/locale/locales/zh-TW/messages.po [hunk 8]
```diff
@@ -4071,9 +4128,9 @@ msgstr "糟糕！"
 msgid "Oh no! Something went wrong."
 msgstr "糟糕！發生了一些錯誤。"
 
-#: src/components/dialogs/nuxs/TenMillion/index.tsx:175
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:224
 msgid "Oh no! We weren't able to generate an image for you to share. Rest assured, we're glad you're here 🦋"
-msgstr ""
+msgstr "糟糕！我們無法產生圖像供您分享。請放心，我們仍然很高興您持續使用藍天 🦋"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:4133]:
```suggestion
msgstr "抱歉！我們無法產生可供你分享的圖片。不過，我們仍然很高興您願意持續使用 Bluesky 🦋"
```

File: src/locale/locales/zh-TW/messages.po [hunk 9]
```diff
@@ -5186,10 +5243,14 @@ msgstr "要求發佈前提供替代文字"
...
 msgstr "登入時要求電子郵件驗證碼"
 
-#: src/screens/Signup/StepInfo/index.tsx:132
+#: src/screens/Signup/StepInfo/index.tsx:156
 msgid "Required for this provider"
 msgstr "此供應商要求必填"
 
+#: src/components/LabelingServiceCard/index.tsx:75
+msgid "Required in your region"
+msgstr ""
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5252]:
```suggestion
msgstr "您所在的區域必填"
```

Update Translates

File: src/locale/locales/zh-TW/messages.po [hunk 10]
```diff
@@ -5715,13 +5776,13 @@ msgstr "仍然分享"
 msgid "Share feed"
 msgstr "分享動態源"
 
-#: src/components/dialogs/nuxs/TenMillion/index.tsx:621
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:678
 msgid "Share image externally"
-msgstr ""
+msgstr "外部分享圖片"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5781]:
```suggestion
msgstr "將圖片分享至外部"
```

File: src/locale/locales/zh-TW/messages.po [hunk 11]
```diff
@@ -6234,9 +6301,9 @@ msgstr "文字輸入框"
 msgid "Thank you. Your report has been sent."
 msgstr "謝謝，您的檢舉已提交。"
 
-#: src/components/dialogs/nuxs/TenMillion/index.tsx:593
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:647
 msgid "Thanks for being one of our first 10 million users."
-msgstr ""
+msgstr "感謝您成為我們首批 1000 萬用戶之一。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:6306]:
```suggestion
msgstr "感謝您成為我們的前 1000 萬用戶之一。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 12]
```diff
@@ -7279,6 +7351,10 @@ msgstr "我們將使用這些資訊來協助訂製您的體驗。"
 msgid "We're having network issues, try again"
 msgstr "我們遇到網路問題，請再試一次"
 
+#: src/components/dialogs/nuxs/NeueTypography.tsx:55
+msgid "We're introducing a new theme font, along with adjustable font sizing."
+msgstr "我們引入了一種新的主題字體，以及字體大小的客製化。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:7356]:
```suggestion
msgstr "我們推出了一款新的主題字體，並加入了字體大小的調整功能。"
```

**Comment by auroursa** [comment]
Thanks @cirx1e as always.
And this PR is now ready to be merged.

File: src/locale/locales/zh-TW/messages.po [hunk 13]
```diff
@@ -6828,37 +6830,38 @@ msgstr "取消對所有 {displayTag} 貼文的靜音"
...
+
+#: src/view/screens/ProfileList.tsx:550
 msgid "Unpin moderation list"
 msgstr "取消釘選內容管理列表"
 
-#: src/view/screens/ProfileList.tsx:346
+#: src/view/screens/ProfileList.tsx:347
 msgid "Unpinned from your feeds"
-msgstr "已從您的動態源取消釘選"
+msgstr "從您的動態源中取消釘選"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:6862]:
```suggestion
msgstr "已從您的動態源中取消釘選"
```


https://github.com/bluesky-social/social-app/blob/8894944d461b0ded19c84aeb38629964f69913c0/src/view/screens/ProfileList.tsx#L344-L348
同 L4488，這部份應該是提示訊息 (toast notifications)

File: src/locale/locales/zh-CN/messages.po [hunk 14]
```diff
@@ -6815,33 +6830,38 @@ msgstr "取消隐藏所有 {displayTag} 帖文"
...
+
+#: src/view/screens/ProfileList.tsx:550
 msgid "Unpin moderation list"
 msgstr "取消固定限制列表"
 
-#: src/view/screens/ProfileList.tsx:346
+#: src/view/screens/ProfileList.tsx:347
 msgid "Unpinned from your feeds"
-msgstr "从你的资讯源中取消固定"
+msgstr "从你的资讯源取消固定"
```
[the remark this answers is not in the document]
Comment (auroursa) [reply src/locale/locales/zh-CN/messages.po:6862]: 谢谢！

**Review by estrattonbailey — APPROVED** [review]
Thank you!

**Pull request merged**